.dotnet/examples/Chat/Example07_StructuredOutputsAsync.cs

```diff
@@ -0,0 +1,60 @@
+﻿using NUnit.Framework;
+using OpenAI.Chat;
+using System;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace OpenAI.Examples;
+
+public partial class ChatExamples
+{
+    [Test]
+    public async Task Example07_StructuredOutputsAsync()
```

Comment:
For consistency, include a sync sample too.

Reply:
Can do that -- separate followup topic for later about why we do and whether it makes sense for us to have the sample structure we do.